install/helm/openchoreo-control-plane/templates/default-resources/wait-for-controller-hook-sa.yaml

```diff
@@ -5,7 +5,7 @@ kind: ServiceAccount
 metadata:
   name: controller-wait-sa
   annotations:
-    "helm.sh/hook": post-install
```

Comment:
Shall we add `helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded` to these resources as well. This is because we don't want to keep these after the helm. chart is successfully installed.